### ISSUE TITLE
Jailbreak History reporting bugs

### ISSUE THREAD

**Issue by Leeful** [hippie68/hippie68.github.io#4]
Ive found a couple of bugs in the kernel exploit success rate history logging.

When you run GoldHEN the kernel exploit success logging works fine but I have noticed that after the kernel has been patched, if you run a regular payload on its own it messes up the accuracy of the reported "kernel exploit success rate: %"

Payloads that require the browser to be left running in the background (e.g dumper or FTP) report a jailbreak failed and other regular payloads (eg backup) report a jailbreak success. Both of which either add or subtract to the success rate percentage of the kernel exploit.

Because there is no need to run the kernel exploit again if the kernel has already been successfully patched I think it would be a good idea to exclude the kernel exploit success logging (or the jb.js exploit ) if a payload is just run on its own to avoid this error.

Another bug is that sometimes instaed of a kernel panic or Jailbreak Failed alert the kernel exploit appears to have run successfully but instead the PS4 freezes and the controller goes unresponsive. This is also logged incorrectly as a successful jailbreak. Not sure how easy it will be to catch this error but I thought I'm mention it.

BTW, Excellent job on the menu. Always good to see new ideas and features like the ones you have added to this.
Keep up the good work. :)

**Comment by hippie68** [comment]
Thank you. I should have fixed those bugs now, hopefully without introducing new ones. As the history log is static text, most changes will become apparent in future log entries.

**Comment by Leeful** [comment]
Thanks for the update. I did a quick test and its working well now. Thanks :)

BTW your FTP(1337) payload does not work on 7.55. I tested it both before and after running GoldHEN and it didn't work.
It says it loads the payload but there is no FTP popup notification and no FTP server running on 1337.

If it helps, this is the FTP payload I use on 7.55 and it also works fine if GoldHEN is already loaded.
[755_FTP_1337.zip](https://github.com/hippie68/hippie68.github.io/files/6542509/755_FTP_1337.zip)
